drivers/storage/portworx/components_test.go

```diff
@@ -13633,6 +13639,329 @@ func TestDeleteNodePodDisruptionBudget(t *testing.T) {
 
 }
 
+func TestUpdateNodePodDisruptionBudgetParallelEnabled(t *testing.T) {
```

Comment:
These are good set of UTs, but I don't see every condition being covered. Eg. Decommissioned nodes, px node status not being okay, non-quorum member nodes cordoned, etc.

Let's go through the code that you have added line by line and make sure we have a test case of each condition. One case even for an ORed and ANDed sub-condition.

Reply:
Updating PDB for decomissioned and non quorum members might not be a good UT as we do not include them in the request to the filterNodes API. Since we are mocking that API is there any way to test that we aren't sending those nodes to the API ?

Reply:
added UTs for - 

- decommissioned node
- px node status is down
- non quorum member also cordoned
- node without PDB cordoned
- node version empty
- When feature enabled, invalid minAvailable value, reconcile 2 times
- when feature disabled, invalid minAvailable, reconcile 2 times